src/app/dataset-view/additional-components/settings-component/settings.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="custom-container px-4 p-responsive">
+    <div class="Layout-sidebar">
+        <div class="pr-md-4 pr-0">
+            <div>
+                <nav aria-label="User settings" data-view-component="true" class="NavigationList">
+                    <ul
+                        role="list"
+                        aria-label="User settings"
+                        nav_classes="org-sub-menu"
+                        data-view-component="true"
+                        class="ActionList"
```

Comment:
I strongly recommend not to use cameCase for class names,
please rename to `action-list`